appserver/admingui/cluster/src/main/resources/org/glassfish/cluster/admingui/Strings_es_ES..properties

```diff
@@ -0,0 +1,439 @@
+cluster.number.instance.running={0} instancias se están ejecutando {1}
+cluster.number.instance.notRunning={0} instancias están paradas {1}
+cluster.number.instance.requireRestart={0} instancias requieren reiniciarse {1}
+cluster.InstancesTitle=Instancias de servidor en cluster
+cluster.InstancesTitleHelp=Crear y administrar instancias de Payara Server en cluster para el cluster actual.
+cluster.instancesTableTitle=Instancias de servidor
+
+cluster.ResourcesTitle=Recursos
+cluster.ResourcesTitleHelp=Todas las instancias en un cluster tienen el mismo conjunto de recursos, resultando el mismo espacio de nombres JNDI.
+cluster.resourcesTableTitle=Recursos
+
+cluster.status.unknown=desconocido
+
+
+cluster.ApplicationsTitle=Aplicaciones
+cluster.ApplicationsTitleHelp=Desplegar, cerrar, habilitar, deshabilitar y borrar las aplicaciones seleccionadas asociadas con el cluster. Borrar la aplicación sólo la borra de este cluster. La aplicación no se cierra.
+clusterApplications.TableTitle=Aplicaciones desplegadas
+
+clusterInstanceNew.PageTitle=New Clustered Server Instance
+clusterInstanceNew.PageTitleHelp=Add a Payara Server instance to the current cluster. The new instance inherits its configuration from the parent cluster.
+clusterInstance.PageTitle=General Information
+instanceName=Instance Name:
+clusterInstance.healthStatus=Health Information:
+
+# this is not implemented.
+#health.status.NOT_RUNNING=No en ejecución
+#health.status.RUNNING=Ejecutando desde
+#health.status.REJOINED=Unificado desde
+#health.status.FAILURE=En fallo desde
+#health.status.SHUTDOWN=Apagado desde
+
+#cluster.ColInstancesStopped=Instancias paradas
+#cluster.ColInstancesRestart=Instancias que requieren reinicio
+#
+#cluster.ColNodeAgent=Agente de nodo
+#cluster.ColInstanceName=Nombre de la instancia
+
+##cluster.numStopped=Instancias paradas
+#cluster.numRunning=Instancias ejecutándose
+#cluster.NewClusteredInstance=Nueva instancia en cluster
+#cluster.startLB=Habilitar balanceo de carga
+#cluster.quiesce=Inactividad...
+#
+
+### Standalone instance
+standaloneInstances.PageTitle=Instancias de Payara Server
+standaloneInstances.PageTitleHelp=Crear y administrar instancias de Payara Server. Instancias en ejecución no pueden borrarse..
+standaloneInstances.TableTitle=Instancias de servidor
+standalone.ColWeight=Peso LB
+
+standaloneInstanceNew.PageTitle=Nueva instancia de Payara Server
+standaloneInstanceNew.PageTitleHelp=Crear una nueva instancia de Payara Server.
+standaloneInstancePropertiesPageTitleHelp=
+
+standalone.ApplicationsTitle=Aplicaciones
+standalone.ApplicationsTitleHelp=Despliegue, cerrar, habilitar, deshabilitar y borrar las aplicaciones selecconadas asociadas con la instancia. Borrar la aplicación solo borra la aplicación de esta instancia. La aplicación no será cerrada.
+standalone.InstanceName=Nombre de la instancia:
+#standalone.tab.Resources=Recursos
```

Comment:
Why are these commented out?

Reply:
Because they are commented on the original file